inference-engine/tests/functional/plugin/conformance/subgraphs_dumper/matchers_cfg.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Adding of config example would be good

Reply:
I'll remove this file. Config parsing functionality not implemented yet. And anyway, I propose an approach that configuration file should be generated by MatchersManager and each certain Matcher. I consider Heath it will be much more convenient for the users to generate default config  specifying special command line option and then modify it if required.